psi4/driver/driver.py

```diff
@@ -1709,6 +1765,73 @@ def frequency(name, **kwargs):
         return core.get_variable('CURRENT ENERGY')
 
 
+def vibanal_wfn(wfn, hess=None, irrep=None, molecule=None, project_trans=True, project_rot=True):
+    # TODO should go back to private
+    import numpy as np
+
+    if hess is None:
```

Comment:
This is a great idea!  Being able to analyze anything passed in is really cool.

Reply:
The `vibanal_wfn` was a minimum distance hookup btwn the driver and the python. There's another `vibanal_str` in `tests/python/vib_analysis` that gets at the analysis from another angle. I expect these functions to evolve a lot as we see what's really needed. The `isotopes` test case shows Hessian reuse (finally!).